src/include/cart/api.h

```diff
@@ -1946,6 +1946,17 @@ int crt_group_secondary_modify(crt_group_t *grp, d_rank_list_t *sec_ranks,
 			d_rank_list_t *prim_ranks, crt_group_mod_op_t op,
 			uint32_t version);
 
+/**
+ * Remove a client id from the Server hash table of Hg and Fabric Interface
+ * addresses.
```

Comment:
I would prefer to not get into details of what we do, so remove references to hg/itnerface here.

Just make it something like crt_cleanup_client_id(). In notes to this api you can mention that for now it is only to be called when psm2 is used and client exists abruptly